src/transformers/integrations/integration_utils.py

```diff
@@ -803,6 +803,10 @@ def setup(self, args, state, model, **kwargs):
         if self._wandb is None:
             return
         self._initialized = True
+
+        # prepare to handle potential configuration issues during setup
+        from wandb.sdk.lib.config_util import ConfigError as WandbConfigError
+        
```

Comment:
Hi @ZIYU-DEEP Could you remove the space at the very beginning of line 809 of this file src/transformers/integrations/integration_utils.py:809, It should solve the issue.

Reply:
thanks so much @pengwei715 for catching this!